mvpa2/base/node.py

```diff
@@ -281,7 +281,9 @@ def set_postproc(self, node):
     def __str__(self, *args, **kwargs):
         return _str(self, *args, **kwargs)
 
-    def __repr__(self, prefixes=[]):
+    def __repr__(self, prefixes=None):
+        if prefixes is None:
+            prefixes = []
```

Comment:
Speed accuracy trade-off. The fix is not that it was broken before. The fix is the function signature that looks more trustworthy now.